### PR TITLE
Add new gated paper.

### PR DESCRIPTION
**Pull request by czotti**

### CODE REVIEW

File: deep-learning/_posts/2017-09-21-Gated-feedback-refinement-network.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    venue: "CVPR 2017"
+    
+pdf: "http://www.cs.umanitoba.ca/~ywang/papers/cvpr17.pdf"
+---
+
+## Model
+
+This paper is a follow up of [Label Refinement Network](https://arxiv.org/pdf/1703.00551v1.pdf).
+The model is based on a encoding-decoding architecture with skip connections, the encoding is done by a VGG-16 network where only the convolutional layers are used.
+The novelty of their work is the use of gating mechanism in the skip connection.
```
Comment (jodoin) [inline deep-learning/_posts/2017-09-21-Gated-feedback-refinement-network.md]: of a gating

File: deep-learning/_posts/2017-09-21-Gated-feedback-refinement-network.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+![](/deep-learning/images/gfrn/fig2.png)
+
+As you can see they use multiple crossentropy loss for each resolution of feature maps on the decoding part. This help the network to extract better information at each resolution.
```
Comment (jodoin) [inline deep-learning/_posts/2017-09-21-Gated-feedback-refinement-network.md]: losses

File: deep-learning/_posts/2017-09-21-Gated-feedback-refinement-network.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+## Gate Refinement unit
+
+The **Gate unit** merge two resolution of feature maps by upsampling the lower resolution and make an element wise product of the features. These features are finally handled by the **Refinement unit** that produce a label segmentation map.
```
Comment (jodoin) [inline deep-learning/_posts/2017-09-21-Gated-feedback-refinement-network.md]: merges
produces

File: deep-learning/_posts/2017-09-21-Gated-feedback-refinement-network.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+The main idea behind this is that lower resolution label maps integrate higher-frequency details and that lead to better segmentation maps.
+
+![](/deep-learning/images/gfrn/fig3.png){: style="width: 365px"}
+![](/deep-learning/images/gfrn/fig4.png){: style="width: 365px"}
+
+## Results
+
+They test their method on three dataset namely, CamVid, Pascal VOC 2012, and a subset of Pascal VOC 2012 with only horses and cows where the task is to separate each part of the animal.
```
Comment (jodoin) [inline deep-learning/_posts/2017-09-21-Gated-feedback-refinement-network.md]: datasets

File: deep-learning/_posts/2017-09-21-Gated-feedback-refinement-network.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+![](/deep-learning/images/gfrn/fig2.png)
+
+As you can see they use multiple crossentropy loss for each resolution of feature maps on the decoding part. This help the network to extract better information at each resolution.
+
+## Gate Refinement unit
+
+The **Gate unit** merge two resolution of feature maps by upsampling the lower resolution and make an element wise product of the features. These features are finally handled by the **Refinement unit** that produce a label segmentation map.
+
+The main idea behind this is that lower resolution label maps integrate higher-frequency details and that lead to better segmentation maps.
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-09-21-Gated-feedback-refinement-network.md]: Are you sure this sentence is right? How can a lower resolution can contain higher-frequency details? It should be the opposite, no?

Comment (czotti) [reply deep-learning/_posts/2017-09-21-Gated-feedback-refinement-network.md]: you are right

**Pull request merged**